addons/mrp/static/src/js/mrp_field_one2many_with_copy.js

```diff
@@ -1,75 +0,0 @@
-/** @odoo-module **/
-
-import { FieldOne2Many } from 'web.relational_fields';
-import ListRenderer from 'web.ListRenderer';
-import fieldRegistry from 'web.field_registry';
-import {_t} from 'web.core';
-
-//----------------------------------------------------
-
-var MrpFieldOne2ManyWithCopyListRenderer = ListRenderer.extend({
-
-    /**
-     * @override
-     */
-    init: function (parent, state, params) {
-        this._super.apply(this, arguments);
-        this.creates.push({
-            string: parent.nodeOptions.copy_text,
-            context: '',
-        });
-    },
-});
-
-var MrpFieldOne2ManyWithCopy = FieldOne2Many.extend({
-
-    /**
-     * @override
-     */
-    init: function (parent, name, record, options) {
-        this._super.apply(this, arguments);
-        this.nodeOptions = _.defaults(this.nodeOptions, {
-            copy_text: _t('Copy Existing Operations'),
-        });
-    },
-    /**
-     * @override
-     */
-    _getRenderer: function () {
-        if (this.view.arch.tag === 'tree') {
-            return MrpFieldOne2ManyWithCopyListRenderer;
-        }
-        return this._super.apply(this, arguments);
-    },
-    /**
-     * @override
-     */
-    _openFormDialog: function (params) {
-        if (params.context === undefined) {
-            return this._super.apply(this, arguments);
-        }
-        const parent = this.getParent();
-        const parentIsNew = parent.state.res_id === undefined;
-        const parentHasChanged = parent.state.isDirty();
-        if (parentIsNew || parentHasChanged) {
-            this.displayNotification({ message: _t('Please click on the "save" button first'), type: 'danger' });
```

Comment:
does it work as expected on a new record? I mean, is the record correctly saved, and then the method called with the correct id if used in context?